bin/lerna.js

```diff
@@ -1,11 +1,12 @@
 #!/usr/bin/env node
+"use strict";
```

Comment:
I got tired of my editor yelling at me with lint errors whenever I opened this file. Easy enough to start linting it automatically, as we no longer support node < v4.